src/app/static/js/editor/script.js

```diff
@@ -282,3 +403,7 @@ $("#download").click(function(e){
     e.preventDefault();
     saveTextAsFile();
 });
+/* alert before leaving the page */
+window.onbeforeunload = function (e) {
+    return "Are you sure you want to leave. All the changes will be lost. You can either download the XML document or submit changes for review.";
```

Comment:
Minor grammar:

`Are you sure you want to leave.` -> `Are you sure you want to leave?`